src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,6 +42,8 @@ public RobotContainer() {
             },
             driveSubsystem));
 
+    StripSubsystem.getInstance().setDefaultCommand(new AllianceColor());

```

Comment:
instead of setting it as the default Command (which will cause the command to run every cycle), we really just need to run it once, right? You could have it run in `robotInit()`

Reply:
That would be good but the plan was for when any other command involving the lights involved the lights it would override this default command. When the command using the lights was done I want the lights to return to just being the team color.